automatic/selenium.powershell/legal/VERIFICATION.txt

```diff
@@ -0,0 +1,17 @@
+VERIFICATION
+Verification is intended to assist the Chocolatey moderators and community
+in verifying that this package's contents are trustworthy.
+
+The module has been downloaded from <https://www.powershellgallery.com/packages/Selenium>
+and can be verified by following these steps:
+
+1. Save the module from the PowerShell Gallery:
+  - Save-Module -Name Selenium -Version 3.0.1
```

Comment:
```suggestion
  - Save-Module -Name Selenium -MaximumVersion 3.0.1
```

Just noticed that -Version isn't actually a valid argument in the PS5/7 sessions I have open.

Reply:
Good catch. It's also missing the required `-Path`. I've updated both of these.